.github/workflows/doc-lint-test.yaml

```diff
@@ -231,7 +282,29 @@ jobs:
       - name: Install geoips documentation packages
         run: pip install -e .[doc]
 
+      - name: Build release notes from yaml files
+        id: release-note-generator
+        run: |
+          update_this_file="$(cat $BASE_GEOIPS_PATH/update_this_release_note)"
+          release_notes_dir="$(dirname $update_this_file)/$(basename $update_this_file .rst)"
+          release_version="$(basename $update_this_file .rst)"
+          echo "Running brassy on directory: $release_notes_dir"
+          brassy --release-version $release_version --no-rich --output-file $update_this_file $release_notes_dir
+          echo "release_note_file=$update_this_file" >> $GITHUB_OUTPUT
+          cat $update_this_file
+      - name: Pinken release note
+        run: pink ${{ steps.release-note-generator.outputs.release_note_file }}
+
+      - name: Upload release note
+        uses: actions/upload-artifact@v4
+        with:
+          name: Release Note
+          path: ${{ steps.release-note-generator.outputs.release_note_file }}
+          if-no-files-found: error
+          overwrite: true
```

Comment:
Is `overwrite: true` needed here? If so, why?

Reply:
No, not really. If this was run with an os matrix, this would prevent errors and only keep the last copy. Happy to keep or remove

Reply:
I find it a little confusing in the absence of os matrix. Maybe remove for now?